src/utility/validation.js

```diff
@@ -191,6 +191,30 @@ const GET_EVENTS_VALIDATION_RULES = [
         .withMessage('publisherId must be a valid id'),
 ];
 
+const CREATE_FUND_VALIDATION_RULES = [
+    body('title').exists().isString().isLength({ max: 50 }).withMessage('title is required'),
+    body('content').exists().isString().isLength({ max: 1000 }).withMessage('content is required'),
+    body('deadline').exists().isDate().withMessage('deadline is required'),
+    body('targetFund').exists().isInt().withMessage('targetFund is required'),
+    body('categories')
+        .exists()
+        .isArray({ min: 1 })
+        .withMessage('categories must be an unempty array')
+        .custom((array) =>
+            array.every(
+                (category) =>
+                    isString(category) &&
+                    variables.CATEGORIES.includes(category)
+            )
+        )
+        .withMessage('categories must be an array of valid categories'),
+    body('city').optional().isString()
```

Comment:
Why is the city optional? 
The address should be checked as follows, which ensures that no required value is missing: 
```
    body('address')
        .exist()
        .isObject()
        .custom((address) => {
            return (
                isString(address.addressLine) &&
                isString(address.city) &&
                isString(address.country)
            );
        })
        .withMessage(
            'address must be an object that contains street, city, and country properties which all must be strings'
        )
```